numba_cuda/numba/cuda/memory_management/nrt.cu

```diff
@@ -78,6 +78,39 @@ extern "C" __device__ void NRT_MemInfo_call_dtor(NRT_MemInfo* mi)
   NRT_MemInfo_destroy(mi);
 }
 
+static void __device__
+nrt_varsize_dtor(void *ptr, size_t size, void *info) {
+    if (info) {
+        /* call element dtor */
+        typedef void dtor_fn_t(void *ptr);
+        dtor_fn_t *dtor = (dtor_fn_t *)info;
+        dtor(ptr);
+    }
+    NRT_Free(ptr);
+}
+
+__device__ NRT_MemInfo* NRT_MemInfo_new_varsize(size_t size)
```

Comment:
```suggestion
static NRT_MemInfo* __device__ NRT_MemInfo_new_varsize(size_t size)
```

Can this also be `static` since it's only called by `NRT_MemInfo_new_varsize_dtor`? It's also not declared in the corresponding `nrt.cuh` header file, so it looks like a private function.